examples/torch/maml_ppo_half_cheetah_dir.py

```diff
@@ -51,21 +51,19 @@ def maml_ppo_half_cheetah_dir(ctxt, seed, epochs, episodes_per_task,
                                               hidden_nonlinearity=torch.tanh,
                                               output_nonlinearity=None)
 
-    max_episode_length = 100
```

Comment:
add this to GymEnv below

Reply:
I think both GymEnvs need it specified here, correct?